pkg/operator/config/config.go

```diff
@@ -50,6 +55,12 @@ type OperatorControllerConfig struct {
 	// List of hooks should be run on the storage classes.
 	// No informers here, because StorageClassController does not accept any.
 	StorageClassHooks []csistorageclasscontroller.StorageClassHookFunc
+
+	// ExtraReplacements defines additional replacements that should be made to assets
+	ExtraReplacementsFunc func() []string
```

Comment:
It could go either way, I am happy to change it.

Reply:
Assuming there are feature gates available in `client.Clients`, this can be a static array populated in `GetAzureDiskOperatorControllerConfig`. The I guess it's file to have just `[]string` here, it may get handy for other CSI drivers.